### ISSUE TITLE
opendkim-keygen: variable to allow 2048/4096 key instead of the default 1024

### ISSUE THREAD

**Issue by jeanmonet** [sunfoxcz/ansible-dkim#9]
Hi, I have noticed that `opendkim-keygen` runs with the default parameter for the RSA key length, which is [1024 bits](https://www.opendkim.org/opendkim-genkey.8.html):
https://github.com/FoxyRoles/ansible-dkim/blob/108f8c6f460aac488d37a03c0f909e6dec8c46ea/tasks/opendkim_keys.yml#L15-L19

The suggestion is to include a variable that allows the user to set for example `-b {{ dkim_opendkim_rsa_keylen }}` to 2048/4096 bits. Also, it may be worth considering the possibility of going for [EdDSA](https://www.mailhardener.com/kb/how-to-use-dkim-with-ed25519) instead of RSA in the future.

**Comment by foxycode** [comment]
@jeanmonet I agree. Your PR is very welcome. Also I would use 2048 bit as default for sure.

**Comment by jeanmonet** [comment]
> @jeanmonet I agree. Your PR is very welcome. Also I would use 2048 bit as default for sure.

As a side note, after some testing (on Ubuntu 20.04 LTS), I think that `opendkim-keygen` generates 2048 keys by default, although the documentation mentions 1024. In any case, it's better to be explicit IMO and have a variable for this, which also allows the use of 4096 bit keys.
